### DISCUSSION TITLE
PowMR Inverter to PACE BMS

### DISCUSSION THREAD

**Discussion by YAYASIMPSON** [ai-republic/bms-to-inverter#29] [Ideas]
I have PowMR Hybrid Inverter and a PACE BMS with 52v 200Ah battery i would like to help to have a result with your superb work.

**Comment by ai-republic** [comment]
Hi @YAYASIMPSON,
I've researched a little bit about the PowMR Hybrid Inverter. It has CAN a communication port which is already great. I guess that the PowMR inverter will probably support the Pylon CAN protocol - which most of the chinese inverters copied.
The PACE BMS implementation I would need to research. Could you tell me which BMS model you have?

**Comment by YAYASIMPSON** [comment]
hello thank's to answer me.
i found this doc i don't know if it's the good for you see file pace.
i have this doc for helo you for powmr see file pow
and my bms is PACE BMS 16S 48V 200Ah

thank's a lot i hope i can helo you with files

Le dim. 19 mai 2024, 03:18, Torsten Oltmanns ***@***.***> a
écrit :

**Reply by ai-republic** [reply]
I think you forgot to attach the files ;)

**Comment by YAYASIMPSON** [comment]


**Reply by ai-republic** [reply]
Hi,
you did not attach files on this Github discussion.

**Comment by YAYASIMPSON** [comment]
[PACE-BMS-CAN-communication-protocol-20161216.pdf](https://github.com/ai-republic/bms-to-inverter/files/15373166/PACE-BMS-CAN-communication-protocol-20161216.pdf)
[POW-HVM5_1_22_translate RS232(2023-09-14 05_44_28).docx](https://github.com/ai-republic/bms-to-inverter/files/15373167/POW-HVM5_1_22_translate.RS232.2023-09-14.05_44_28.docx)
sorry 🤣 i post by gmail and i put files but no in github 🤣

**Comment by YAYASIMPSON** [comment]
hello it's good the files or no? i can test with my material (POWMR 10.2K
Hybrid Inverter and battery with PACE BMS 16S 48V 200Ah)

Can you send me protocol to send in RASPI 3B

thanks a lot


Le lun. 20 mai 2024 à 06:01, Torsten Oltmanns ***@***.***> a
écrit :

> Hi,
> you did not attach files on this Github discussion.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ai-republic/bms-to-inverter/discussions/29#discussioncomment-9492132>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AW56KNPSUWERVCWZ44YUMG3ZDFYRLAVCNFSM6AAAAABH3T6IMWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TIOJSGEZTE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>


-- 
¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤
Yannick CHALOT
52 Sente du Pré, 95320 Saint leu la forêt
tel : 06.08.71.05.36
@ : ***@***.***
¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤

**Comment by ai-republic** [comment]
Yes, sorry I haven't had time to look at it since I'm very busy at the moment. I will try to look at it next weekend

**Comment by ai-republic** [comment]
I've implemented the PACE BMS CAN binding. If you could test it that would be great. Just download the new _Configurator_ app and choose the PACE_BMS binding and NONE as inverter. Please also set the Log-Level to debug. That way we can see we get from the BMS and helps me to fix any issues.

**Comment by YAYASIMPSON** [comment]
ok super no problem i test it when i can and reply you thanks a lot

Le lun. 10 juin 2024, 08:17, Torsten Oltmanns ***@***.***> a
écrit :

**Comment by ai-republic** [comment]
@YAYASIMPSON  did you get a chance to test it yet? Would be nice to hear if everything worked out ;)

**Reply by AlexK98** [reply]
Hello, there is no CAN port on powmr 10.2, it has RS485 port for comms with BMS. And new versions of inverter have option to choose between pace-modbus 485 and pylon 485 protocols.

**Reply by ai-republic** [reply]
Then the _PYLON_RS485_ inverter binding would be the choice to go with
